index.html

```diff
@@ -0,0 +1,132 @@
+                    </div>
+                    <div class="indicator"><span></span></div>
+
+                    <div class="main-program-content">
+                        <div class="program one">
+                            <img src="images/speech.png" alt="speech logo">
+                            <h3>Speech</h3>
+                            <p>Get a chance to talk to designers and tech founders, ask questions about their journey and inspirations</p>
+                        </div>
+                        <div class="program two">
+                            <img src="images/exhibition.png" alt="speech logo">
+                            <h3>Exhibition</h3>
+                            <p>Get a chance to talk to designers and tech founders, ask questions about their journey and inspirations</p>
+                        </div>
+                        <div class="program three">
+                            <img src="images/forum-icon.png" alt="speech logo">
+                            <h3>Forum</h3>
+                            <p>Get a chance to talk to designers and tech founders, ask questions about their journey and inspirations</p>
+                        </div>
+                        <div class="program four">
+                            <img src="images/workshop.png" alt="speech logo">
+                            <h3>Workshop</h3>
+                            <p>Get a chance to talk to designers and tech founders, ask questions about their journey and inspirations</p>
+                        </div>
+                        <div class="program five">
+                            <img src="images/network-icon.png" alt="speech logo">
+                            <h3>CC Network</h3>
+                            <p>Get a chance to talk to designers and tech founders, ask questions about their journey and inspirations</p>
+                        </div>
+                        <button class="join-btn">Join WRC CAR-NNECT SUMMIT 2023</button>
+                    </div>
+                    
+
+                </section>
+
+                <section class="featured-speakers">
+                   
+
+                </section>
+                <section class="partners" id="hide-desktop">
+                    <div class="partners_top">
+                        <h3>Partners</h3>
+                    </div>
+                    <div class="indicator"><span></span></div> 
+                    <div class="partners-logo">
+                        <img src="images/ford.svg" alt="">
+                        <img src="images/hyundai.svg" alt="">
+                        <img src="images/toyota.svg" alt="">
+                    </div> 
+        
+        
+                </section>
+            </main>
+            <footer class="footer" id="hide-desktop">
+                <img src="images/logo.png" alt="logo" id="logo">
+                <h4>2023 WRC Car-nnect <br>Some Rights Reserved</h4>
+        
+            </footer>
```

Comment:
- [ ] These sections are not visible on your home page. Kindly make them visible as we have them on the guiding design

### Your Design
![your-home-footer](https://user-images.githubusercontent.com/26597432/226091035-b525485f-52f7-43c5-9edc-27e37f3151e0.png)

### Guiding Design
![temp-home-footer](https://user-images.githubusercontent.com/26597432/226091043-bc3957dc-a29a-4cf1-9699-a2a556ac30b2.png)